ServerRoomMonitoring.Api/Program.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+
+namespace ServerRoomMonitoring.Api
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                //.ConfigureLogging(logging =>
```

Comment:
Czemu logger wyłączony? Jak dorzucisz plik log4net.config to będzie działać od razu z taką konfiguracją i będzie można wstrzykiwać logger.